engine/context/src/main/java/io/deephaven/engine/context/QueryCompiler.java

```diff
@@ -697,13 +697,29 @@ private void maybeCreateClassHelper(String fqClassName, String finalCode, String
 
         final StandardJavaFileManager fileManager = compiler.getStandardFileManager(null, null, null);
 
-        final boolean result = compiler.getTask(compilerOutput,
-                fileManager,
-                null,
-                compilerOptions,
-                null,
-                Collections.singletonList(new JavaSourceFromString(fqClassName, finalCode)))
-                .call();
+        boolean result = false;
+        boolean exceptionThrown = false;
+        try {
+            result = compiler.getTask(compilerOutput,
+                    fileManager,
+                    null,
+                    compilerOptions,
+                    null,
+                    Collections.singletonList(new JavaSourceFromString(fqClassName, finalCode)))
+                    .call();
+        } catch (final Exception err) {
```

Comment:
Would you really want to suppress an `Error`? Probably better to catch `Throwable`, given what's happening here.